interfaces/automated-testing/scd/observation.yaml

```diff
@@ -0,0 +1,473 @@
+            to reject operational intent update notifications originating from a USS that does not manage
+            the operational intent.
+        uss_availability:
+          $ref: '#/components/schemas/UssAvailabilityState'
+        version:
+          type: integer
+          format: int32
+          example: 1
+          description: |-
+            Numeric version of this operational intent which increments upon each change in the operational intent,
+            regardless of whether any field of the operational intent reference changes.  A USS with the
+            details of this operational intent when it was at a particular version does not need to retrieve
+            the details again until the version changes.
+        state:
+          $ref: '#/components/schemas/OperationalIntentState'
+        ovn:
+          description: |-
+            Opaque version number of this operational intent.  Populated only when the OperationalIntentReference
+            is managed by the USS retrieving or providing it.  Not populated when the
+            OperationalIntentReference is not managed by the USS retrieving or providing it (instead, the
+            USS must obtain the OVN from the details retrieved from the managing USS).
+          anyOf:
+          - $ref: '#/components/schemas/EntityOVN'
+        time_start:
+          description: Beginning time of operational intent.
+          anyOf:
+          - $ref: '#/components/schemas/Time'
+        time_end:
+          description: End time of operational intent.
+          anyOf:
+          - $ref: '#/components/schemas/Time'
+        uss_base_url:
+          $ref: '#/components/schemas/OperationalIntentUssBaseURL'
+        subscription_id:
+          description: |-
+            The ID of the subscription that is ensuring the operational intent manager receives relevant
+            airspace updates.
+          anyOf:
+          - $ref: '#/components/schemas/SubscriptionID'
+
+    OperationalIntentUssBaseURL:
+      description: |-
+        The base URL of a USS implementation that implements the parts of the USS-USS API necessary for
+        providing the details of this operational intent, and telemetry during non-conformance or contingency,
+        if applicable.
+      anyOf:
+      - $ref: '#/components/schemas/UssBaseURL'
+
+    OperationalIntent:
+      description: Full description of a UTM operational intent.
+      required:
+      - reference
+      - details
+      type: object
+      properties:
+        reference:
+          $ref: '#/components/schemas/OperationalIntentReference'
+        details:
+          $ref: '#/components/schemas/OperationalIntentDetails'
+
+        
+    OperationalIntentList:
+      description: A list of operational intent 
+      type: array
+      items: 
+        $ref: '#/components/schemas/OperationalIntent'
+
+paths:
+  /scd_qualifier_data/{id}:
+    get:
+      parameters:
```

Comment:
We need to specify where we want to look for operational intents; I would expect a Volume4D as one of the parameters in the request body.  RID qualifier uses only a 2D area in its observation request because that corresponds to how users would likely query a Display Provider.  For strategic coordination, the altitude and time bounds are more likely to be relevant.